packages/rum/src/domain/record/serialize.ts

```diff
@@ -401,13 +416,26 @@ function getAttributesForPrivacyLevel(
   /**
    * Serialize the scroll state for each element only for full snapshot
    */
-  if (serializationContext === 'full-snapshot') {
-    if (element.scrollLeft) {
-      safeAttrs.rr_scrollLeft = Math.round(element.scrollLeft)
-    }
-    if (element.scrollTop) {
-      safeAttrs.rr_scrollTop = Math.round(element.scrollTop)
-    }
+  let scrollPositions
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏I think using two independent variables here would be easier on the bundle size (no repeated property access, no `?.` operator):

```suggestion
  let scrollTop: number | undefined
  let scrollLeft: number | undefined
```